Makefile

```
@@ -21,7 +21,7 @@ ARCH_ALIAS_x86_64 = amd64
 ARCH_ALIAS_aarch64 = arm64
 ARCH_ALIAS = $(shell echo "$(ARCH_ALIAS_$(ARCH))")
 
-QEMU_VERSION=v8.1.5
+QEMU_VERSION=v9.2.2
```

Please use the full version tag; the `v9.2.2` may be reused for a later build:

```suggestion
QEMU_VERSION=v9.2.2-52
```